packages/ckeditor5-engine/src/view/editableelement.ts

```diff
@@ -42,6 +42,13 @@ export default class EditableElement extends ObservableMixin( ContainerElement )
 	 */
 	declare public isFocused: boolean;
 
+	/**
+	 * Placeholder of current editable element.
```

Comment:
"**current** editable" isn't fitting in this context.

By using `EditableElement` instance you're working only with that particular editable element. Yo're working only with this instance. There's no relation to other editables.

Also you could add a code listing in here showing how to change it in runtime, including the entire call chain, like you did in sample: `editor.editing.view.document.getRoot( 'main' ).placeholder = 'New placeholder';`.